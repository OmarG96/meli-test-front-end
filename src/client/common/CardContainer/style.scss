@import "../../utils/scss/variables.scss";

.card-container-root {
  grid-column: 2 / span 10;
  background-color: $white;
  border-radius: 4px;
  max-width: 920px;
  width: 100%;
  margin: 0 auto 5% auto;
}
