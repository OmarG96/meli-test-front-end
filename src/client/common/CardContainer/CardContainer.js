import React from "react";
import { node } from "prop-types";

import "./style.scss";

const CardContainer = ({ children }) => {
  return <div className="card-container-root">{children}</div>;
};

CardContainer.propTypes = {
  children: node,
};

export default CardContainer;
