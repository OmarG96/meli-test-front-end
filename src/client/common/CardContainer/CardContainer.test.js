import React from "react";
import CardContainer from ".";
import { render } from "@testing-library/react";

describe("GIVEN CardContainer", () => {
  describe("WHEN is rendered", () => {
    it("THEN match to snapshot", () => {
      const { asFragment } = render(
        <CardContainer>
          <div>example</div>
        </CardContainer>
      );
      expect(asFragment()).toMatchSnapshot();
    });
  });
});
