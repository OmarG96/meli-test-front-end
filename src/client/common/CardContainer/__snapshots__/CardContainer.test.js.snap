// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GIVEN CardContainer WHEN is rendered THEN match to snapshot 1`] = `
<DocumentFragment>
  <div
    class="card-container-root"
  >
    <div>
      example
    </div>
  </div>
</DocumentFragment>
`;
